src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -268,6 +272,7 @@ public void requestNotFoundForUnknownId() {
     assertThat(getResponse.getStatusCode(), is(HttpURLConnection.HTTP_NOT_FOUND));
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
What part of this method produces unchecked warnings?

Reply:
The `assertThat(requests, hasItems(`;

Reply:
Ok, thanks